eslint.config.js

```diff
@@ -1,40 +1,30 @@
 import comments from "@eslint-community/eslint-plugin-eslint-comments/configs";
 import eslint from "@eslint/js";
 import vitest from "@vitest/eslint-plugin";
-import eslintPlugin from "eslint-plugin-eslint-plugin";
 import jsdoc from "eslint-plugin-jsdoc";
 import jsonc from "eslint-plugin-jsonc";
 import markdown from "eslint-plugin-markdown";
 import n from "eslint-plugin-n";
-import packageJson from "eslint-plugin-package-json/configs/recommended";
+import packageJson from "eslint-plugin-package-json";
 import perfectionist from "eslint-plugin-perfectionist";
 import * as regexp from "eslint-plugin-regexp";
 import yml from "eslint-plugin-yml";
 import tseslint from "typescript-eslint";
 
 export default tseslint.config(
 	{
-		ignores: [
-			"**/*.snap",
-			"coverage",
-			"lib",
-			"node_modules",
-			"pnpm-lock.yaml",
-			"docs/rules/*/*.ts",
-			".*rc.*",
```

Comment:
https://github.com/JoshuaKGoldberg/create-typescript-app/issues/1936, in that `.eslint-doc-generatorrc.cjs` will need to be ignored.